eslint/config.js

```diff
@@ -39,7 +40,6 @@ module.exports = {
 		'comma-spacing': 'error',
 		'comma-style': 'error',
 		'computed-property-spacing': [ 'error', 'always' ],
-		'constructor-super': 'error',
```

Comment:
`computed-property-spacing` is also es6 specific. Is there a way to identify whart also belong to es6?

Reply:
The ESLint rule list has a ES6 category of rules: https://eslint.org/docs/rules/#ecmascript-6

Reply:
Ah okey, that explains everything 😃